mmv1/third_party/terraform/tests/resource_compute_backend_service_test.go.erb

```diff
@@ -1400,6 +1400,40 @@ resource "google_compute_http_health_check" "zero" {
 }
 `, serviceName, checkName)
 }
+
+func testAccComputeBackendService_withCdnPolicyBypassCacheOnRequestHeaders(serviceName, checkName string) string {
```

Comment:
This test isn't callable -- can you add a testing function for it further up in the file? And because the field itself is updatable, can that testing function also include an update step? Thanks!

Reply:
done. I added two steps to TestAccComputeBackendService_withCdnPolicy to set the bypass cache headers.